pymess/migrations/0026_migration.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 3.1 on 2022-01-07 10:44
+
+from django.db import migrations, models
+import django.db.models.deletion
+import pymess.models.emails
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('pymess', '0025_migration'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='attachment',
+            name='filename',
+            field=models.CharField(blank=True, max_length=100, null=True),
+        ),
+        migrations.CreateModel(
+            name='EmailTemplateAttachment',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False)),
```

Comment:
ty generuješ migrace z projektu viď to asi nemůžeš jelikož ten ti tu odstraní verbose name a tady v knihovně by to být mělo. Ostatním to bude řvát, že nejsou aktuální migrace. Možná sem ty verbose name asi ručně doplň aby sis to zjednodušil (anglicky)

Reply:
Už tam jsou.